1-reql-language/asynchronous.md

```diff
@@ -21,7 +21,7 @@ No special procedures or commands are necessary to execute RethinkDB queries asy
 
 In addition, RethinkDB's cursors and feeds implement an [EventEmitter interface][ee] compatible with Node's. This allows your application to set up listeners to receive data from queries as the data becomes available.
 
-[ee]: /api/javascript/event_emitter-cursor/
+[ee]: /api/javascript/event_emitter/
```

Comment:
This is actually the wrong page. It should be `/api/javascript/ee-cursor/`.
Note that one is for connections, and the other one is for cursors. We want the cursor one here.